packages/frontend/src/contexts/AppContext/AppContext.tsx

```diff
@@ -69,17 +62,17 @@ const AppContextProvider: FC = ({ children }) => {
   }, [provider])
 
   const sdk = useMemo(() => {
-    return new Hop(reactAppNetwork, provider?.getSigner())
+    return new Hop(hopAppNetwork, provider?.getSigner())
```

Comment:
This should be the original value for `REACT_APP_NETWORK` but I see it's being assigned here

![DeepinScreenshot_select-area_20220115095425](https://user-images.githubusercontent.com/168240/149632433-c3de2a2b-c25d-4014-b689-d51da943500a.png)

The reason for this is if `REACT_APP_NETWORK` is set to `staging` then the sdk should also be initialized with the `staging` "network" because it uses the [staging](https://github.com/hop-protocol/hop/blob/develop/packages/core/src/addresses/staging.ts) set of contracts from core package

Reply:
ahhh, this makes a lot of sense. thanks for the insight @miguelmota